src/sardana/macroserver/test/res/macros/testmacros.py

```diff
@@ -0,0 +1,148 @@
+from sardana.macroserver.macro import Type, Macro
+
+MSG_TEMP = "Parsing failed (result: %r; expected: %r)"
```

Comment:
I would say that decoding failed. Or better "Parsing or decoding failed..."

Reply:
Solved at [commit b31a5b3](https://github.com/sardana-org/sardana/pull/781/commits/b31a5be9ebfbb9d9d5226dc0ab3e395cc52468c2)